akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala

```diff
@@ -1725,43 +1725,50 @@ private[stream] object Collect {
 
       val onPushWhenBufferFull: () => Unit = strategy match {
         case EmitEarly =>
-          () => {
-            if (!isTimerActive(timerName))
-              push(out, buffer.dequeue()._2)
-            else {
-              cancelTimer(timerName)
-              onTimer(timerName)
+          () =>
+            {
+              if (!isTimerActive(timerName))
+                push(out, buffer.dequeue()._2)
+              else {
+                cancelTimer(timerName)
+                onTimer(timerName)
+              }
+              grabAndPull()
             }
-            grabAndPull()
-          }
         case _: DropHead =>
-          () => {
-            buffer.dropHead()
-            grabAndPull()
-          }
+          () =>
+            {
+              buffer.dropHead()
+              grabAndPull()
+            }
         case _: DropTail =>
-          () => {
-            buffer.dropTail()
-            grabAndPull()
-          }
+          () =>
+            {
+              buffer.dropTail()
+              grabAndPull()
+            }
         case _: DropNew =>
-          () => {
-            grab(in)
-            pull(in)
-          }
+          () =>
+            {
+              grab(in)
+              pull(in)
+            }
         case _: DropBuffer =>
-          () => {
-            buffer.clear()
-            grabAndPull()
-          }
+          () =>
+            {
+              buffer.clear()
+              grabAndPull()
+            }
         case _: Fail =>
-          () => {
-            failStage(BufferOverflowException(s"Buffer overflow for delay operator (max capacity was: $size)!"))
-          }
+          () =>
+            {
+              failStage(BufferOverflowException(s"Buffer overflow for delay operator (max capacity was: $size)!"))
+            }
         case _: Backpressure =>
-          () => {
-            throw new IllegalStateException("Delay buffer must never overflow in Backpressure mode")
-          }
+          () =>
+            {
+              throw new IllegalStateException("Delay buffer must never overflow in Backpressure mode")
+            }
```

Comment:
I have no idea what ScalaFMT is up to, formats differently in IntelliJ on save than from SBT now ;(